source/index.rst

```diff
@@ -1,41 +1,43 @@
-.. PySCF documentation master file, created by
-   sphinx-quickstart on Thu Jan 15 01:55:04 2015.
+
+.. PySCF index file, created by `ablog start` on Mon Feb  8 20:10:55 2021.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to PySCF documentation!
-===============================
- 
-PySCF is a collection of electronic structure programs powered by Python.
-The package aims to provide a simple, light-weight, and efficient platform for
-quantum chemistry calculations and code development.  The program is developed
-with the following principles:
-
-* Easy to install, to use, to extend and to be embedded;
+Welcome to the PySCF Website!
+=================================
 
-* Minimal requirements on libraries (no Boost or MPI) and computing
-  resources (perhaps sacrificing efficiency to reduce I/O);
+The Python-based Simulations of Chemistry Framework
```

Comment:
This should be a title, no?

Reply:
I don't think so because the title is already in the header of the website so that would show up twice on the same page right next to each other